### PR TITLE
feat: storyblok cms

### PR DESCRIPTION
**Pull request by WojtekTheWebDev**
Add storyblok, adjust home page to the page content

## Description

Add storyblok to the project, make home more realted with the products on the page 

## Related Issue

BC-27

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: packages/theme/nuxt.config.js [hunk 1]
```diff
@@ -78,11 +78,16 @@ export default {
...
+    '@nuxtjs/markdownit',
     'nuxt-i18n',
     'cookie-universal-nuxt',
-    'vue-scrollto/nuxt',
-    '@vue-storefront/middleware/nuxt'
+    'vue-scrollto/nuxt'
   ],
+  markdownit: {
+    runtime: true
+  },
```
Comment (Mateuszpietrusinski) [inline packages/theme/nuxt.config.js:90]: What this flag doing?

File: packages/theme/nuxt.config.js [hunk 2]
```diff
@@ -78,11 +78,16 @@ export default {
   ],
   // Modules: https://go.nuxtjs.dev/config-modules
   modules: [
+    '@vue-storefront/storyblok/nuxt',
+    '@vue-storefront/middleware/nuxt',
+    '@nuxtjs/markdownit',
```
Comment (Mateuszpietrusinski) [inline packages/theme/nuxt.config.js:83]: same what is the purpose of that module?

File: packages/theme/pages/Home.vue [hunk 3]
```diff
@@ -146,136 +101,121 @@ export default defineComponent({
...
+      search,
+      loading
+    } = useProduct('home-products');
+
+    onSSR(async () => {
+      await search({
+        limit: 5,
+        include: 'options,variants'
+      });
+    });
```
Comment (Mateuszpietrusinski) [inline packages/theme/pages/Home.vue:127]: instead of onSRR use useAsync from nuxt

File: packages/theme/pages/_id.vue [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  },
+  setup() {
+    const { error: nuxtError } = useContext();
+    const route = useRoute();
+    const slug = computed(() => route.value.params.id);
+    const { search, content: page, loading, error } = useContent('unique-id');
+
+    onSSR(async () => {
+      await search({ url: slug.value });
+    });
```
Comment (Mateuszpietrusinski) [inline packages/theme/pages/_id.vue:35]: please change onSSR to useAsync

**Review by Mateuszpietrusinski — CHANGES_REQUESTED** [review]
Small adjustments needed

File: packages/theme/nuxt.config.js (hunk 2 above)
Comment (WojtekTheWebDev) [reply packages/theme/nuxt.config.js:83]: [That module ](https://www.npmjs.com/package/@nuxtjs/markdownit) <- module allows to parse markdown into html code (used in the TextBox cms component which contains markdown text)

File: packages/theme/nuxt.config.js (hunk 1 above)
Comment (WojtekTheWebDev) [reply packages/theme/nuxt.config.js:90]: Flag enables to use `$md` in templates

File: packages/theme/layouts/error.vue [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+  <div id="error">
+    <SfImage
+      class="image"
+      :src="addBasePath('/error/error.svg')"
+      alt="leaves"
+      :width="412"
+      :height="412"
+    />
+    <SfHeading
+      :title="error.statusCode === 404 ? 'Page not found' : 'An error occured'"
```
Comment (scernisevs) [inline packages/theme/layouts/error.vue:11]: could we get all the prop strings passed to the heading component translated please? :)

**Review by scernisevs — COMMENTED** [review]
Looks good with 1 minor comment. Also a question, do we want to keep the static assets on the repo or should we upload them to the CMS?

**Review by scernisevs — APPROVED** [review]

**Review by Mateuszpietrusinski — APPROVED** [review]

**Pull request merged**